src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpVarReducer.Rewriter.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Utilities;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Simplification;
+
+namespace Microsoft.CodeAnalysis.CSharp.Simplification
+{
+    internal partial class CSharpVarReducer
+    {
+        private class Rewriter : AbstractReductionRewriter
+        {
+            public Rewriter(ObjectPool<IReductionRewriter> pool)
+                : base(pool)
+            {
+            }
+
+            private SyntaxNode ProcessTypeSyntax(TypeSyntax typeSyntax)
+            {
+                this.CancellationToken.ThrowIfCancellationRequested();
```

Comment:
❔ Any particular reason why we have all the `this.` qualifications?

Reply:
it's a personal preference.  i find that it makes it totally clear if you're accessing the instance or static side of something

Reply:
Thanks, just wanted to make sure it wasn't a new simplification failure.